src/js/events/__tests__/MetronomeClient-test.ts

```diff
@@ -0,0 +1,270 @@
+const mockRequest = jest.fn();
```

Comment:
Jest usually hoists all its stuff so that we can have `import` at the top doesn't it?

Reply:
Yes and no, it hoists the `jest.mock` call, but not the constant, therefore I get this:


```log
 FAIL  src/js/events/__tests__/MetronomeClient-test.ts
  ● Test suite failed to run

    ReferenceError: mockRequest is not defined

      4 | import { Observable } from "rxjs/Observable";
      5 |
    > 6 | import Config from "../config/Config";
```